Upstream-Status: Pending [Not submitted to upstream yet]
Signed-off-by: Vishnu Banavath <vishnu.banavath@arm.com>

From 5fdf7792ead29784718d30d8a8a5383a994e2fef Mon Sep 17 00:00:00 2001
From: Vishnu Banavath <vishnu.banavath@arm.com>
Date: Thu, 2 Dec 2021 12:51:11 +0000
Subject: [PATCH] plat-corstone1000: increase OPTEE core heap size

This change is to increase optee core heap size to 131072
from its default value to fit openAMP and smm-gateway
in SEL0

Signed-off-by: Vishnu Banavath <vishnu.banavath@arm.com>

diff --git a/core/arch/arm/plat-corstone1000/conf.mk b/core/arch/arm/plat-corstone1000/conf.mk
index 7a4aa13c..65d652a4 100644
--- a/core/arch/arm/plat-corstone1000/conf.mk
+++ b/core/arch/arm/plat-corstone1000/conf.mk
@@ -31,6 +31,7 @@ CFG_DDR_SIZE ?= 0x80000000
 CFG_DT_ADDR ?= 0x82100000
 CFG_DTB_MAX_SIZE ?= 0x100000
 
+CFG_CORE_HEAP_SIZE ?= 131072
 $(call force,CFG_PSCI_ARM64,y)
 $(call force,CFG_DT,y)
 $(call force,CFG_EXTERNAL_DTB_OVERLAY,y)
-- 
2.17.1

